tests/settings/e2e/features/stakepool-settings.feature

```diff
@@ -0,0 +1,34 @@
+@e2e
+Feature: Stake-pool server settings
+
+  Background:
+    Given I have completed the basic setup
+    And I have the following wallets:
+      | name        |
+      | Test Wallet |
+
+  Scenario: Navigating to Stake-pool settings screen and selecting custom stake-pool server
+    Given I am on the General Settings "stake-pools" screen
+    When custom server is the default option
```

Comment:
This step is not very clear. In case another option is selected, is it supposed to select the Custom option? If so, it should instead click the dropdown, then click the Custom option, rather than select the input element